lightning/src/routing/router.rs

```diff
@@ -198,7 +300,10 @@ pub fn get_route<L: Deref>(our_node_id: &PublicKey, network: &NetworkGraph, targ
 
 	let mut targets = BinaryHeap::new(); //TODO: Do we care about switching to eg Fibbonaci heap?
 	let mut dist = HashMap::with_capacity(network.get_nodes().len());
+	let recommended_available_msat = final_value_msat * 4;
```

Comment:
Why *4? If we anticipate moving towards mostly using `htlc_maximum_msat` (and notes set that reasonably conservatively), then we should generally be able to hit `htlc_maximum_msat`? Or are we anticipating that nodes won't be conservative with it?

Reply:
Not sure what this has to do with `htlc_maximum_msat`. Here I'm just recommending to collect 4 times the amount we're transferring, so that later we have a better variety of paths to select from.